crowbar_framework/app/views/backup/index.html.haml

```diff
@@ -0,0 +1,68 @@
+.container
+  .row
+    .col-xs-12
+      %h2
+        = t(".title")
+.container
+  .row
+    .panel.panel-default
+      .panel-heading
+        %button.btn.btn-primary{ data: { toggle: "modal", target: "#create_backup_modal" } }
+          = t(".create_backup_image")
+        .btn.btn-primary.pull-right{ data: { toggle: "modal", target: "#upload_modal" } }
+          = t(".upload_image")
+      %table.table
+        %thead
+          %tr
+            %th
+              = t(".image_name")
+            %th
+              = t(".created_at")
+            %th
+            %th
+        %tbody
+          - @backups.each do |backup|
+            %tr
+              %td
+                = link_to backup_download_path(backup.name, backup.created_at) do
+                  = icon_tag(:cloud_download, backup.name)
+              %td
+                = date_parse(backup.created_at)
```

Comment:
That should be done by the I18n helpers, e.g. `l(backup.created_at, format: :short)`.